sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Relation.scala

```diff
@@ -56,7 +56,7 @@ case class DataSourceV2Relation(
 
   lazy val (
```

Comment:
// afterScanFilters: predicates that need to be evaluated after the scan.
// pushedFilters: predicates that will be pushed down and only evaluated in the underlying data sources.

Reply:
they can overlap(think about parquet row group filter), so I'd remove the `only` in the above comment.

Reply:
uh... Then, we should mention it in the comment too. For example, 
```
// afterScanFilters: predicates that need to be evaluated after the scan.
// pushedFilters: predicates that will be pushed down and only evaluated in the underlying data sources.
// Note: afterScanFilters and pushedFilters can overlap. For example, the parquet row group filter
```